livemaps-api/APIFunctions/AtlasUnits.cs

```diff
@@ -1,34 +1,39 @@
 using System;
-using System.IO;
+using System.Net.Http;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Azure.WebJobs;
 using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.AspNetCore.Http;
 using Microsoft.Extensions.Logging;
 using Newtonsoft.Json;
-using System.Net.Http;
-using ssir.api.Models.Atlas;
+using Ssir.Api.Models.Atlas;
 
-namespace ssir.api.APIFunctions
+namespace Ssir.Api.APIFunctions
 {
     public static class AtlasUnits
     {
         [FunctionName("AtlasUnits")]
         public static async Task<IActionResult> Run(
+            //This HTTP trigger sends a client HTTP request through the atlasSubscriptionKey and atlasDataSetId parameters to get featureIds.
```

Comment:
When I mentioned comments, my intent was to walk developers unfamiliar with the service and the end-to-end scenario through the high-level flow of what the sample does and to call out why we're doing what we're doing in the areas that are tricky and may be unclear. 

Unfortunately, the comments that were added offer little value.   They simply state the obvious of what the next line is doing , which the code itself tells us.   For example, in this area...   what is an "atlas data set id and what does it do for us?"

Reply:
For that matter, what is Atlas?

Reply:
@jsquire I've updated the comments, please review.